pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -46,15 +46,26 @@ func (admitter *PodEvictionAdmitter) Admit(ar *admissionv1.AdmissionReview) *adm
 	if err != nil {
 		return denied(fmt.Sprintf("kubevirt failed getting the vmi: %s", err.Error()))
 	}
-	if !migrations.VMIMigratableOnEviction(admitter.ClusterConfig, vmi) {
+
+	evictionStrategy := migrations.VMIEvictionStrategy(admitter.ClusterConfig, vmi)
```

Comment:
I prefer the current way it's written because it's more explicit and precise about what eviction strategies should mark the VMI.

FOr example, if we add a new eviction strategy "EvictionStrategySomethingNew" then the code as I've written it continues to operate within it's current scope without automatically expanding to marking the VMIs when the new `EvictionStrategySomethingNew` is used, which might be a surprise